staging/src/k8s.io/kubectl/SECURITY_CONTACTS

```diff
@@ -0,0 +1,17 @@
+# Defined below are the security contacts for this repo.
+#
+# They are the contact point for the Product Security Committee to reach out
+# to for triaging and handling of incoming issues.
+#
+# The below names agree to abide by the
+# [Embargo Policy](https://git.k8s.io/security/private-distributors-list.md#embargo-policy)
+# and will be removed and replaced if they violate that agreement.
+#
+# DO NOT REPORT SECURITY VULNERABILITIES DIRECTLY TO THESE NAMES, FOLLOW THE
+# INSTRUCTIONS AT https://kubernetes.io/security/
+
+cjcullen
+jessfraz
```

Comment:
I think should be s/jessfraz/joelsmith. Ref: https://github.com/kubernetes/community/tree/master/committee-product-security#members

(probably needs update for all staging repos, but that can be done in another PR IMO)

cc @liggitt

Reply:
> (probably needs update for all staging repos, but that can be done in another PR IMO)

created https://github.com/kubernetes/kubernetes/pull/78470